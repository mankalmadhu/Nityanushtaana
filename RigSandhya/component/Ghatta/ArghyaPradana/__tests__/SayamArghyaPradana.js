import React from 'react';
import ಸಾಯಮರ್ಘ್ಯ from '../SayamArghyaPradana';
import Renderer from 'react-test-renderer';
 
jest.mock('../../Ghatta.vinyasa');
jest.mock('../../../Vishaya/Vishaya',()=>'ವಿಷಯ');
jest.mock('../../../Vishaya/Purvanga',()=>'ಪೂರ್ವಾ೦ಗ');
jest.mock('../../../Vishaya/Viniyoga',()=>'ವಿನಿಯೋಗ');
jest.mock('../../Gayatri/Trihruti',()=>'ತ್ರಿಹೃತಿ');
jest.mock('../../Gayatri/GayatriMantra',()=>'ಗಾಯತ್ರೀಮ೦ತ್ರ');
jest.mock('../../Sankalpa/Oomapo',()=>'ಓಮಾಪೋ');
jest.mock('../KalaTeetaArgyaPradana',()=>'ಕಾಲಾತೀತಅರ್ಘ್ಯ');

test("ಸಾಯಮರ್ಘ್ಯ", () => {
   expect(Renderer.create(
    <ಸಾಯಮರ್ಘ್ಯ />
  ).toJSON()).toMatchSnapshot();

});