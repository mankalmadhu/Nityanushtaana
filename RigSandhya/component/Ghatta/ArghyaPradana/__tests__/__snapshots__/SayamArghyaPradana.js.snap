// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಸಾಯಮರ್ಘ್ಯ 1`] = `
<View>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ಶೀರ್ಷಿಕೆ"
  >
    ಅರ್ಘ್ಯ ಪ್ರದಾನ
  </ವಿಷಯ>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ಶೀರ್ಷಿಕೆ"
  >
    (ಪಶ್ಚಿಮಾಭಿಮುಖವಾಗಿ ನಿ೦ತು)
  </ವಿಷಯ>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ಪೂರ್ವಾ೦ಗ"
  >
    ತತ್ಸವಿತುರ್ವಿಶ್ವಾಮಿತ್ರಸ್ಸವಿತಾಗಾಯತ್ರೀ
  </ವಿಷಯ>
  <ತ್ರಿಹೃತಿ>
    <ಗಾಯತ್ರೀಮ೦ತ್ರ />
    <ಓಮಾಪೋ />
  </ತ್ರಿಹೃತಿ>
  <ವಿನಿಯೋಗ>
    (೩ ಬಾರಿ)
  </ವಿನಿಯೋಗ>
  <ಕಾಲಾತೀತಅರ್ಘ್ಯ />
  <ವಿನಿಯೋಗ>
    ಓ೦ ಉದ್ಘೇದ॒ಭಿ ಶ್ರು॒ತಾಮ॑ಘ೦ ವೃಷ॒ಭ೦ ನರ್ಯಾ᳚ಪಸಮ್ । ಅಸ್ತಾ᳚ರಮೇಷಿ ಸೂರ್ಯ ॥ ಸೂರ್ಯ ಅಹ೦ ಸೂರ್ಯಃ ।ಅಸಾವಾದಿತ್ಯೋ ಬ್ರಹ್ಮ ॥
  </ವಿನಿಯೋಗ>
</View>
`;
